### PR TITLE
Update README

### LINKED ISSUE #17: Update Project Website README Content

### PR DESCRIPTION
**Pull request by tayian**
Fixes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+* our **team project** for the course **CS2103T** in AY2023/2024 Semester 1.
```
Comment (geraldngjx) [inline README.md]: Would it be possible to capitalize the "o" of "our"?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
+* This is based on the `AddressBook-Level3` project created by the [SE-EDU initiative](https://se-education.org).
+* The project is a desktop application used for organising contact and company details.
```
Comment (geraldngjx) [inline README.md]: Could you edit the description to reflect the standard description we have used in the User Guide? For reference, "Connectify is a networking platform designed to help professionals efficiently manage their connections and enhance their networking experiences."

**Review by geraldngjx — CHANGES_REQUESTED** [review]
Looks good! Just a few small changes to make!

**Comment by tayian** [comment]
Thanks Gerald for the heads up! Will edit accordingly

File: README.md (hunk 1 above)
Comment (tayian) [reply README.md]: Thanks Gerald for the heads up! Will edit accordingly

**Review by geraldngjx — APPROVED** [review]
LGTM!

**Pull request merged**